tests/forte/forte_backbone_test.py

```diff
@@ -56,3 +56,10 @@ def test_import_models(self):
 
     def test_import_trainer(self):
         from forte.trainer.base import BaseTrainer
+
+    def test_import_forte_modules(self):
```

Comment:
I think you also want to import them in their short form, for example, should be `from forte import pipeline`

Reply:
corrected.